web/client/epics/map.js

```diff
@@ -61,7 +66,45 @@ const handleCreationLayerError = (action$, store) =>
         ]) : Rx.Observable.empty();
     });
 
+const startEditing = (action$, store) =>
+    action$.ofType(FEATURE_EDITING)
```

Comment:
use an intermediation between the "geometry editing service" provided by the draw support and the editing that the featuregrid triggers.